src/js/media/media.js

```diff
@@ -81,6 +75,15 @@ vjs.MediaTechController.prototype.initControlsListeners = function(){
   });
 };
 
+vjs.MediaTechController.prototype.removeControlsListeners = function () {
```

Comment:
There's already [a removeControlsListeners](https://github.com/benjipott/video.js/blob/hotfix/switch-between-tech/src/js/media/media.js#L126). Should this be combined with that?